azure/table_azure_app_service_web_app.go

```diff
@@ -286,3 +292,25 @@ func getAppServiceWebAppSiteAuthSetting(ctx context.Context, d *plugin.QueryData
 
 	return op, nil
 }
+
+//// TRANSFORM FUNCTION
+
+func webAppIdentity(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	data := d.HydrateItem.(web.Site)
+	objectMap := make(map[string]interface{})
+	if data.Identity != nil {
+		if data.Identity.Type != "" {
```

Comment:
What is the type of `data.Identity.Type`? Can't we do a `nil` check, instead of empty string (`""`)?